### PR TITLE
MC-15985: Add payment documentation

### PR DESCRIPTION
**Pull request by GuillaumeVialle**
#### Changes made
- Added /payments documentation

#### Related PRs
- UI https://github.com/cloudops/cloudmc-ui/pull/1969
- Core: https://github.com/cloudops/cloudmc-core/pull/1583
- Core DoD fix: https://github.com/cloudops/cloudmc-core/pull/1589

### CODE REVIEW

File: source/includes/administration/_payments.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+Retrieve a list of payment processed for an organization.
+
+Query Params | &nbsp;
+---- | -----------
+`organizationId`<br/>*UUID* | The ID of the organization for which we want to see the payments.
+
+Attributes | &nbsp;
+------- | -----------
+`id`<br/>*UUID* | A payment's unique identifier.
+`invoice`<br/>*Object* | The invoice associated to the payment.
```
Comment (fyrnaga) [inline source/includes/administration/_payments.md:75]: If we do return some info from this object, we should document the as well.

File: source/includes/administration/_payments.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+`paymentReference`*String* | Payment reference send to the payment provider to indentify the transaction.
+`paymentTraceReference`*String* | Additional reference used to find a payment on the payment provider side.
+`errorMsg`<br/>*String* | The error message returned by the payment provider if the payment failed.
+`totalAmount`<br/>*BigDecimal* | The total amount of the invoice charged on the card.
+`taxAmount`<br/>*BigDecimal* | The tax amount that was included in the charge.
+`confirmationNumber`<br/>*String* | Confirmation number return returned from the payment provider.
+`deleted`<br/>*String* | Whether the payment has been deleted. Payment is deleted if it was unsucessful and the transaction not found on the payment provider side.
```
Comment (fyrnaga) [inline source/includes/administration/_payments.md]: There is currently no way to retrieve non delete information. We should not return it. I would just remove it from doc.

Comment (GuillaumeVialle) [reply source/includes/administration/_payments.md]: I just realized it, for the organizations we just don't include it in the doc so I'll remove it.

Comment (GuillaumeVialle) [reply source/includes/administration/_payments.md]: 19dfb704627e60df77aa1efc04074a3ba35aaaab and 9fe36c37810df74d81631c9709ae25a615fa0a50

File: source/includes/administration/_payments.md (hunk 1 above)
Comment (GuillaumeVialle) [reply source/includes/administration/_payments.md:75]: 57d243c2ec46528da14373e7f6b587671ef577b7

**Review by fyrnaga — APPROVED** [review]

File: source/includes/administration/_payments.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+Retrieve a payment.
+
+Attributes | &nbsp;
+------- | -----------
+`id`<br/>*UUID* | A payment's unique identifier.
+`invoice`<br/>*Object* | The invoice associated to the payment.
+`invoice.id`<br/>*UUID* | The UUID of the invoice.
+`invoice.invoiceId`<br/>*String* | The human readable invoice id.
+`invoice.createdDate`<br/>*date* | The created date of the invoice.
+`paymentReference`*String* | Payment reference send to the payment provider to indentify the transaction.
```
Comment (HassanHaidar) [inline source/includes/administration/_payments.md]: payment reference sent* to the payment ...

File: source/includes/administration/_payments.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+`paymentTraceReference`*String* | Additional reference used to find a payment on the payment provider side.
+`errorMsg`<br/>*String* | The error message returned by the payment provider if the payment failed.
+`totalAmount`<br/>*BigDecimal* | The total amount of the invoice charged on the card.
+`taxAmount`<br/>*BigDecimal* | The tax amount that was included in the charge.
+`confirmationNumber`<br/>*String* | Confirmation number return returned from the payment provider.
```
Comment (HassanHaidar) [inline source/includes/administration/_payments.md]: return returned -> returned*

File: source/includes/administration/_payments.md [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+`paymentDate`<br/>*date* | The date on which the payment was send to the provider.
```
Comment (HassanHaidar) [inline source/includes/administration/_payments.md]: send -> sent*

File: source/includes/administration/_payments.md [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+`organizationId`<br/>*UUID* | The ID of the organization for which we want to see the payments.
+
+Attributes | &nbsp;
+------- | -----------
+`id`<br/>*UUID* | A payment's unique identifier.
+`invoice`<br/>*Object* | The invoice associated to the payment.
+`invoice.id`<br/>*UUID* | The UUID of the invoice.
+`invoice.invoiceId`<br/>*String* | The human readable invoice id.
+`invoice.createdDate`<br/>*date* | The created date of the invoice.
+`paymentReference`*String* | Payment reference send to the payment provider to indentify the transaction.
```
Comment (HassanHaidar) [inline source/includes/administration/_payments.md]: send -> sent*

File: source/includes/administration/_payments.md [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+`paymentTraceReference`*String* | Additional reference used to find a payment on the payment provider side.
+`errorMsg`<br/>*String* | The error message returned by the payment provider if the payment failed.
+`totalAmount`<br/>*BigDecimal* | The total amount of the invoice charged on the card.
+`taxAmount`<br/>*BigDecimal* | The tax amount that was included in the charge.
+`confirmationNumber`<br/>*String* | Confirmation number return returned from the payment provider.
```
Comment (HassanHaidar) [inline source/includes/administration/_payments.md]: return returned -> returned*

File: source/includes/administration/_payments.md [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+`paymentDate`<br/>*date* | The date on which the payment was send to the provider.
```
Comment (HassanHaidar) [inline source/includes/administration/_payments.md]: send -> sent*

Comment (GuillaumeVialle) [reply source/includes/administration/_payments.md]: a1c2559aec4c73004740c42c7f8602975f928e5a

File: source/includes/administration/_payments.md (hunk 7 above)
Comment (GuillaumeVialle) [reply source/includes/administration/_payments.md]: a1c2559aec4c73004740c42c7f8602975f928e5a

File: source/includes/administration/_payments.md (hunk 6 above)
Comment (GuillaumeVialle) [reply source/includes/administration/_payments.md]: a1c2559aec4c73004740c42c7f8602975f928e5a

File: source/includes/administration/_payments.md (hunk 5 above)
Comment (GuillaumeVialle) [reply source/includes/administration/_payments.md]: a1c2559aec4c73004740c42c7f8602975f928e5a

File: source/includes/administration/_payments.md (hunk 4 above)
Comment (GuillaumeVialle) [reply source/includes/administration/_payments.md]: a1c2559aec4c73004740c42c7f8602975f928e5a

File: source/includes/administration/_payments.md (hunk 3 above)
Comment (GuillaumeVialle) [reply source/includes/administration/_payments.md]: a1c2559aec4c73004740c42c7f8602975f928e5a

**Review by HassanHaidar — APPROVED** [review]
Thank you for the fix! Looks good 👍

**Pull request merged**